webpack.browser.config.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   resolve: {
     alias: {
       storage$: path.resolve(__dirname, 'src/storage/browser.js'),
+      session$: path.resolve(__dirname, 'src/session/browser.js')
```

Comment:
Don't think it needs it, I actually replace that module for QML. It will start a new session everytime the app starts and that is that.

Reply:
Yes, but we don't want that concepts in app. That's a web only concept and should not exist here.

Reply:
Swaped this for a `noop`, if I don't keep the two separate modules QML will return errors about not having `setTimeout` even though the session code is never touched, since it is still in the bundle.